package.json

```diff
@@ -34,50 +34,47 @@
     "build:babel:esm": "BABEL_ENV=esm babel --extensions \".ts,.tsx\" --copy-files src --out-dir dist/esm",
     "build:types": "tsc",
     "lint": "eslint 'src/**/*.ts?(x)'",
-    "prettier": "prettier --write 'src/**/*.{ts?(x),css}'",
+    "prettier": "prettier --write \"src/**/*.{ts?(x),css}\"",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "deploy-storybook": "gh-pages -d storybook-static"
   },
-  "peerDependencies": {
-    "@material-ui/core": ">=4.11.3",
-    "react": ">=16.8.0",
-    "react-dom": ">=16.8.0"
-  },
   "devDependencies": {
-    "@babel/cli": "^7.12.10",
-    "@babel/core": "^7.13.10",
-    "@babel/plugin-proposal-export-default-from": "^7.12.1",
-    "@babel/plugin-transform-typescript": "^7.12.1",
-    "@material-ui/core": "^4.11.3",
-    "@storybook/addon-actions": "^6.1.21",
-    "@storybook/addon-essentials": "^6.1.21",
-    "@storybook/addon-links": "^6.1.21",
-    "@storybook/react": "^6.1.21",
-    "@types/node": "^14.14.20",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.12.0",
-    "@typescript-eslint/parser": "^4.12.0",
+    "@babel/cli": "^7.14.5",
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-proposal-export-default-from": "^7.14.5",
+    "@babel/plugin-transform-typescript": "^7.14.6",
+    "@emotion/react": "^11.4.0",
```

Comment:
Why is emotion added as a dep?

Reply:
It is a dependency of the new version of `material-ui`

Reply:
In order to support the work of the sample storybook page.